sdks/java/io/hbase/pom.xml

```diff
@@ -52,29 +50,6 @@
               </goals>
               <configuration>
                 <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
why is the shade section still here at all?

Do you still want to shade `protobuf-lite`?